rotkehlchen/api/server.py

```diff
@@ -335,7 +335,7 @@ def __init__(
         self.ws_server: Optional[WebSocketServer] = None
 
         self.flask_app.errorhandler(HTTPStatus.NOT_FOUND)(endpoint_not_found)  # type: ignore
-        self.flask_app.register_error_handler(Exception, self.unhandled_exception)
+        self.flask_app.register_error_handler(Exception, self.unhandled_exception)  # type: ignore
```

Comment:
A bit dissapointed with my "fix". Spent more time than I should have trying to understand why mypy fails here.

It was due to this typing change: https://github.com/pallets/flask/issues/4095

But couldn't make it happy, so I had to ignore.